docs/scripts/clean_dags.py

```diff
@@ -0,0 +1,299 @@
+import re
+import sys
+import networkx as nx
+
+rules_to_hide = [
```

Comment:
would it make sense to change this into a rules to keep? :innocent:

Reply:
yea it would :cry:

Reply:
It's also easier if rules get added later down the line :grimacing: